packages/react-slider/Spec.md

```diff
@@ -196,21 +198,194 @@ _Explain how the component will behave in use, including:_
+
+  return (
+    <div>
+      <RangedSlider defaultValue={[2, 8]} />
+      <RangedSlider value={rangedSliderValue} onChange={sliderOnChange} />
+    </div>
+  );
+};
+```
+
+## Structure
+
+- _**Public**_
+  ```jsx
+  <RangedSlider defaultValue={{ lowerValue: 0, upperValue: 100 }} />
+  ```
+- _**Internal**_
+
+  ```jsx
+  <slots.root {...slotProps.root}>
+    {state.marks && <slots.marksWrapper {...slotProps.marksWrapper} />}
+    <slots.sliderWrapper {...slotProps.sliderWrapper}>
+      <slots.rail {...slotProps.rail} />
+      <slots.trackWrapper {...slotProps.trackWrapper}>
+        <slots.track {...slotProps.track} />
+      </slots.trackWrapper>
+      <slots.lowerThumbWrapper {...slotProps.lowerThumbWrapper}>
+        // The hidden input element is moved inside the thumbWrapper for styling purposes regarding focus
+        <slots.inputLower {...slotProps.inputLower} />
+        <slots.lowerThumb {...slotProps.lowerThumb} />
+      </slots.lowerThumbWrapper>
+      <slots.upperThumbWrapper {...slotProps.upperThumbWrapper}>
+        <slots.inputUpper {...slotProps.inputUpper} />
+        <slots.upperThumb {...slotProps.upperThumb} />
+      </slots.upperThumbWrapper>
+      <slots.activeRail {...slotProps.activeRail} />
+    </slots.sliderWrapper>
+  </slots.root>
+  ```
+
+- _**DOM** - how the component will be rendered as HTML elements_
+
+```jsx
+<div className="ms-RangedSlider-root">
+  <div className="ms-RangedSlider-markContainer">
+    <div className="ms-RangedSlider-markItemContainer">
+      <div className="ms-RangedSlider-mark" />
+      <div className="ms-RangedSlider-markLabel" />
+    </div>
+  </div>
+  <div className="ms-RangedSlider-wrapper">
+    <div className="ms-Slider-rail" />
+    <div className="ms-Slider-trackWrapper">
+      <div className="ms-Slider-track" />
+    </div>
+    <div className="ms-RangedSlider-lowerThumbWrapper">
+      <div className="ms-RangedSlider-inputLower" />
+      <div className="ms-Slider-lowerThumb" />
+    </div>
+    <div className="ms-RangedSlider-upperThumbWrapper">
+      <div className="ms-RangedSlider-inputUpper" />
+      <div className="ms-RangedSlider-upperThumb" />
+    </div>
+    <div className="ms-RangedSlider-activeRail" />
+  </div>
+</div>
+```
+
+## Behaviors
+
+_Explain how the component will behave in use, including:_
+
+- _Component States_
+
+  - **Disabled**
+    - When disabled, all touch and mouse events are ignored, and the RangedSlider's value never updates.
+    - Does not allow focus and is read only.
+  - **Focused**
+    - Focus indicators only appear when keyboard tabbing/directional keystrokes and disappears when the mouse/touch interactions occur. The individual thumb hidden input elements maintain focus.
+
+- _Interaction_
+  - _Keyboard_
+    Handles the same keyboard increment/decrement values from the `Slider` component. When the active thumb surpasses the lower or upper thumb, focus and the set value both switch to the other thumb.
```

Comment:
I remember we talked about asking design if this is the behavior we wanted. Has that conversation happened?

Reply:
Ditto line 386 below.

Reply:
@czearing spoke with design offline and verified that this is what we want so I'm resolving this.